recipe/meta.yaml

```diff
@@ -1,8 +1,6 @@
-{% set version = "0.11.7" %}
-
 package:
   name: ruamel_yaml
-  version: 0.11.7
+  version: 0.11.14
 
 source:
   fn: ruamel_yaml-{{ version }}.tar.gz
```

Comment:
Actually, seems like `{{ version }}` may as well be used in the `package` section too.